internal/replicator/k8s.go

```diff
@@ -273,6 +274,16 @@ func getSecretOnLabel(uuid string, namespace string) (v1.Secret, error) {
 	return secretList.Items[0], nil
 }
 
+func getCurrentNamespace() string {

```

Comment:
maybe we can do something like 
if (env.namespace set; use this)
else if(getcurrentnamespace works; use this)
else use dev-tooling (or smth else)

Reply:
Sounds good to me